### PR TITLE
Upgrade elasticsearch version to 5.3.2

### PR DESCRIPTION
**Pull request by michiomochi**
Elasticsearch version upgrade to 5.3.2.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -75,23 +75,10 @@ jobs:
...
 
-              docker build -t $IMAGE_TAG_DEPLOY image/
-              docker push $IMAGE_TAG_DEPLOY
-
-              docker build -t $LATEST_TAG image/
-              docker push $LATEST_TAG
-
-              git tag $TIMESTAMP
-              git push --tags
+              make publish
```
Comment (michiomochi) [inline .circleci/config.yml:82]: Use `make publish` command.

File: Makefile [hunk 2]
```diff
@@ -1,18 +1,14 @@
-BASETAG=bitjourney/elasticsearch-ci
-TIMESTAMP=`date "+%Y%m%d%H%M"`
-IMAGETAG=$(BASETAG):$(TIMESTAMP)
-LATESTTAG=$(BASETAG):latest
+BASE_TAG:=bitjourney/elasticsearch-ci
+TIMESTAMP:=`date "+%Y%m%d%H%M"`
+TIMESTAMP_TAG:=$(BASE_TAG):$(TIMESTAMP)
+VERSION_TAG:=$(BASE_TAG):5.3.2
```
Comment (michiomochi) [inline Makefile:4]: Change `LATEST_TAG` to `VERSION_TAG`.
If want to use the latest, using latest TIMESTAMP_TAG.

File: Makefile [hunk 3]
```diff
@@ -1,18 +1,14 @@
-BASETAG=bitjourney/elasticsearch-ci
-TIMESTAMP=`date "+%Y%m%d%H%M"`
-IMAGETAG=$(BASETAG):$(TIMESTAMP)
-LATESTTAG=$(BASETAG):latest
+BASE_TAG:=bitjourney/elasticsearch-ci
```
Comment (michiomochi) [inline Makefile:1]: `=` is lazy calling in Makefile. It isn't necessary lazy calling here, so use `:=`.
see: http://qiita.com/takuyachallenge/items/c73b1d2db3761b931109#%E5%8D%98%E7%B4%94%E5%B1%95%E9%96%8B%E5%A4%89%E6%95%B0

**Comment by michiomochi** [comment]
@bitjourney/staff please review 🙏

**Comment by gfx** [comment]
LGTM

**Comment by michiomochi** [comment]
寿司ゆき:thanks

**Pull request merged**